internal/hardware/validator_test.go

```diff
@@ -196,19 +198,49 @@ var _ = Describe("Cluster host requirements", func() {
 		details1, details2   models.ClusterHostRequirementsDetails
 	)
 
+	const (
+		prefixedRequirementsEnv = "MYAPP_" + requirementsEnv
+		versionRequirements     = `[{
+  			"version": "4.6",
+  			"master": {
+  			  "cpu_cores": 4,
+  			  "ram_gib": 16,
+  			  "disk_size_gb": 120
+  			},
+  			"worker": {
+  			  "cpu_cores": 2,
+  			  "ram_gib": 8,
+  			  "disk_size_gb": 120
+  			}
+		}, {
+  			"version": "4.7",
+  			"master": {
+  			  "cpu_cores": 5,
+  			  "ram_gib": 17,
+  			  "disk_size_gb": 121
+  			},
+  			"worker": {
+  			  "cpu_cores": 3,
+  			  "ram_gib": 9,
+  			  "disk_size_gb": 122
+  			}
+		}]`
+	)
+
 	BeforeEach(func() {
 		operatorName1 := "op-one"
 		operatorName2 := "op-two"
 
 		clusterID := strfmt.UUID(uuid.New().String())
 		cluster = &common.Cluster{Cluster: models.Cluster{
-			ID: &clusterID,
+			ID:               &clusterID,
+			OpenshiftVersion: "4.5",
```

Comment:
Assert that `OpenshiftVersion` is not in `versionRequirements` both to clarify intent and make sure that no one breaks this accidentally in the future